docs/fast_data/faq/debezium.md

```diff
@@ -0,0 +1,133 @@
+
+2024-04-17T15:37:57.430055843Z 2024-04-17 15:37:57,429 INFO   ||  [Worker clientId=connect-1, groupId=debezium-test] Disconnecting from node 1 due to request timeout.   [org.apache.kafka.clients.NetworkClient]
+2024-04-17T15:37:57.430390748Z 2024-04-17 15:37:57,430 INFO   ||  [Worker clientId=connect-1, groupId=debezium-test] Cancelled in-flight METADATA request with correlation id 1106 due to node 1 being disconnected (elapsed time since creation: 42006ms, elapsed time since send: 42006ms, request timeout: 40000ms)   [org.apache.kafka.clients.NetworkClient]
+```
+
+These logs typically indicate that idle connections with Kafka brokers are being closed by the Kafka Connect worker and are not a cause for concern.
+
+## Multiple replicas of Debezium Connect
+
+If you are running Debezium in a Kafka Connect cluster, consider the following before adding more than one worker to your cluster:
+
+- A Kafka Connect cluster distributes connector tasks across workers, but a Debezium connector has only one task. Thus, even with multiple workers, only one will be active at a time, while the others remain idle. The benefit is that if a worker fails, the task is automatically reassigned to another healthy worker.
+- Kafka Connect exposes REST APIs that are used extensively to manage the Debezium connector. With multiple workers, if the API call is forwarded to a worker that is not the coordinator, it needs to forward the request to the cluster coordinator. Therefore, you need to ensure that replicas can communicate with each other. In particular, each replica needs the `CONNECT_REST_ADVERTISED_HOST_NAME` environment variable to be configured with its reachable hostname. This is not feasible with common cloud technologies that replicate Docker containers (e.g. Azure App Service) because usually it is not possbile to change the value of an environment variable across replicated containers.
+
+# Debezium Oracle connector
+
+This section addresses questions specific to the Oracle connector.
+
+## How to add the Oracle JDBC to the Debezium Docker image?
+Due to licensing reasons, if you want to use Debezium to connect to an Oracle database, you need to build your own image and manually include the Oracle JDBC. Use the following Dockerfile (ensure the JDBC group, version, and MD5 are correct):
+```Dockerfile
+FROM quay.io/debezium/connect:2.5
+
+ENV KAFKA_CONNECT_JDBC_DIR=$KAFKA_CONNECT_PLUGINS_DIR/kafka-connect-jdbc
+
+# These should point to the driver version to be used
+ENV MAVEN_DEP_DESTINATION=$KAFKA_HOME/libs \
+    ORACLE_JDBC_REPO="com/oracle/database/jdbc" \
+    ORACLE_JDBC_GROUP="ojdbc10" \
+    ORACLE_JDBC_VERSION="19.21.0.0" \
+    ORACLE_JDBC_MD5=55544d916412e364112695485d078a66
+
+# Download Oracle JDBC driver
+RUN docker-maven-download central "$ORACLE_JDBC_REPO" "$ORACLE_JDBC_GROUP" "$ORACLE_JDBC_VERSION" "$ORACLE_JDBC_MD5"
+
+USER kafka
+```
+
+## Oracle connector fails with ORA-00600 error
```

Comment:
also here, it looks more a troubleshooting rather than a faq

Reply:
see previous comment